### PR TITLE
SH-64: create jenkins job

### PR DESCRIPTION
**Pull request by skirsson**

### CODE REVIEW

File: jenkins/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        pollSCM('H/5 * * * *')
+    }
+
+    stages {
+        stage('Build') {
+            agent {
+                docker {
+                    reuseNode true
+                    image 'adoptopenjdk/maven-openjdk11:latest'
+                    args '-v $HOME/.m2:/root/.m2'
```
Comment (mpiacenza-propify) [inline jenkins/Jenkinsfile]: is $HOME the workspace?  seems like there's a workspace variable we should be using here instead.

File: jenkins/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                }
+            }
+            steps {
+                sh "mvn -B -U -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -DskipTests clean package"
```
Comment (mpiacenza-propify) [inline jenkins/Jenkinsfile:34]: oooh interesting.  so this runs inside the docker agent specified above?  nice!  Why skipTests?

Comment (skirsson) [reply jenkins/Jenkinsfile:34]: That's right... yes, I skip them because there are failed tests

Comment (mpiacenza-propify) [reply jenkins/Jenkinsfile:34]: where does the .jar file go?  not seeing it listed as an artifact.  also - maybe a separate process, but likely need a release tag in git.

File: src/main/resources/OH-INF/thing/yale/yrd652_0_0.xml [hunk 3]
```diff
@@ -54,7 +54,7 @@ Touchscreen deadbolt without keyway<br /> <h1>Overview</h1><ul><li>Installs On S
     <!-- DEVICE PROPERTY DEFINITIONS -->
     <properties>
       <property name="vendor">ASSA ABLOY</property>
-      <property name="modelId">YRD256</property>
+      <property name="modelId">YRD652</property>
```
Comment (mpiacenza-propify) [inline src/main/resources/OH-INF/thing/yale/yrd652_0_0.xml:57]: Is this a regression?

**Review by mpiacenza-propify — APPROVED** [review]

File: src/main/resources/OH-INF/thing/yale/yrd652_0_0.xml (hunk 3 above)
Comment (skirsson) [reply src/main/resources/OH-INF/thing/yale/yrd652_0_0.xml:57]: No, it was intentional

**Pull request merged**